docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -31,7 +31,7 @@ gnome-keyring-daemon -r --unlock --components=secrets
 
 :::note
 
-The `gnome-keyring-daemon -r --unlock --components=secrets` command prompts you to specify a password. After you enter the password, press `Ctrl` + `D` twice to continue the terminal session.
+The `gnome-keyring-daemon -r --unlock --components=secrets` command can appear to delay, but it is waiting for you to enter a password. Type your keyring password (typically the same as your Linux user password), then press `Ctrl`+`D` twice to continue the terminal session.
```

Comment:
Would "hang" be better than "delay" here? I think a delay is slow/late response, but what we're experiencing here is an indefinite hang up that will never return unless the user aborts the command.

Reply:
@t1m0thyj: Changing this to "hang" -- wasn't aware of this definition, but now that I am, I agree w/ this. Thank ya!